molecule/connect-scale-up/converge.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: Verify - kafka_connect5 is not deployed yet
+  hosts: kafka-connect2
+  gather_facts: false
+  tasks:
+    - name: Get connectors details
+      uri:
+        url: "http://kafka-connect2:8083/connectors?expand=status"
+        status_code: 200
+        force_basic_auth: true
+      register: connect_cluster_response
+
+    - name: Show the Status response
+      debug:
+        msg: '{{ connect_cluster_response.json["sample-connector-2"].status.tasks }}'
+
+    - set_fact:
+        hosts_all: '{{ connect_cluster_response.json["sample-connector-2"].status.tasks | map(attribute="worker_id") | list }}'
+        hosts_connect5: '{{ connect_cluster_response.json["sample-connector-2"].status.tasks | map(attribute="worker_id") | map("regex_search",".*connect5.*") | select("string") | list  }}'
+    - name: Assert that there are no hosts with connect5 name in workers list
+      assert:
+        that: "hosts_connect5 | length == 0"
+
+- name: Verify - control_center
+  hosts: control_center
+  gather_facts: false
+  tasks:
+    - name: Check line connect cluster2  Should not have connect5 setup yet
+      import_role:
+        name: confluent.test
+        tasks_from: check_property.yml
+      vars:
+        file_path: /etc/confluent-control-center/control-center-production.properties
+        property: confluent.controlcenter.connect.connect-elastic.cluster
+        expected_value: http://kafka-connect2:8083
+
+    - name: Check line connect cluster3 Should not have connect4 setup yet
+      import_role:
+        name: confluent.test
+        tasks_from: check_property.yml
+      vars:
+        file_path: /etc/confluent-control-center/control-center-production.properties
+        property: confluent.controlcenter.connect.connect-ssl.cluster
+        expected_value: https://kafka-connect3:8083
+
+- name: Add new brokers Nodes into in-memory inventory to scale-up
+  hosts: all
+  gather_facts: false
+  tasks:
+    - name: Add new Broker Node kafka-broker4
+      add_host:
+        hostname: kafka-connect4
+        groups:
+          - kafka_connect
+          - ssl
+
+    - name: Add new Broker Node kafka-broker5
+      add_host:
+        hostname: kafka-connect5
+        groups:
+          - kafka_connect
+          - elastic
```

Comment:
Please get rid of un-necessary groups

Reply:
The intent here is to test scale-up with multiple groups of connect.
kafka-connect4 is being added to 1 group and kafka-connect5 is being added to another group (group.id), and validating whether those 2 new nodes are getting added to correct cluster group or not. Also validating that connector-plugins installed on one group should not be there in second group, and vice versa.